src/core/contracts/contract.ts

```diff
@@ -28,7 +29,7 @@ interface ContractResult {
   blockHash: string;
 }
 
-export const DSNPMigrationABI: ethers.utils.ParamType[] = [
+export const DSNP_MIGRATION_ABI: ethers.utils.ParamType[] = [
```

Comment:
if we're already changing this file, my preference is to move constant _values_  to the top, between imports and the first exported function.